test/e2e/getExample.js

```diff
@@ -0,0 +1,93 @@
+const fs = require("fs");
+const path = require("path");
+const playwright = require("@recordreplay/playwright");
+const cli = require("@recordreplay/recordings-cli");
+const _ = require("lodash");
+
+const { recordNode } = require("./recordNode");
+const { elapsedTime, waitUntilMessage } = require("./utils");
+
+async function record(browserName, example) {
+  console.log(`Recording Example:`, example, browserName);
+
+  const browser = await playwright[browserName].launch({
+    headless: false,
+  });
+
+  const context = await browser.newContext();
+  const page = await context.newPage();
+  try {
+    await page.goto(example);
+    await waitUntilMessage(page, "Example__Finished");
+  } catch (e) {
+    console.log("Timed out recording example");
+  } finally {
+    await page.close();
+    await context.close();
+    await browser.close();
+  }
+}
+
+async function upload(state, example) {
+  const list = cli.listAllRecordings();
+  const recording = _.findLast(list, rec => rec.metadata.uri == example);
+
+  console.log(`Uploading example:${example}`, recording.id);
+
+  const recordingId = await cli.uploadRecording(recording.id, {
+    apiKey: state.replayApiKey,
+  });
+
+  console.log(`Uploaded example:${example}`, `${state.APP_SERVER}/recording/${recordingId}`);
+
+  return recordingId;
+}
+
+function updateExampleFile(state, example, recordingId) {
+  const newExampleRecordings = { ...state.ExampleRecordings, [example]: recordingId };
+  fs.writeFileSync("./test/example-recordings.json", JSON.stringify(newExampleRecordings, null, 2));
+}
+
+async function recordExample(state, example, target) {
+  const browser = target == "gecko" ? "firefox" : "chromium";
```

Comment:
This check doesn't make any sense to me. Why do we designate `gecko` in the manifest but then translate it into `firefox` here? Would it be better to just update the manifest to say `firefox` and remove this ternary?

Reply:
Agreed.

I could make the argument for gecko being consistent with chromium, but in a playwright world it is simpler to use playwright conventions instead of mapping between the two.